modules/update-upgrading-web.adoc

```diff
@@ -62,6 +62,11 @@ and click *Save*.
 +
 The Input channel
 *Update status* changes to *Update to <product-version> in progress*, and you can review the progress of the cluster update by watching the progress bars for the Operators and nodes.
++
+[NOTE]
+====
+If you are updating your cluster to the next minor version, it is recommended to verify that your worker nodes are upgraded before deploying workloads that rely on a new feature. Any pools with worker nodes that are not yet updated are displayed on the *Cluster Settings* page.
```

Comment:
[The existing CLI docs][1] remind folks what a minor version bump means, with `like version 4.y to 4.(y+1)`.  I'm ok with "folks will remember without a reminder" and dropping the reminder from both locations or "folks might want a reminder" and including the reminder in both locations, but it seems strange to me to include the reminder in one of two very similar blocks.

[1]: https://github.com/openshift/openshift-docs/blame/a157c88586d18c45772b95d9f64468deabc057c0/modules/update-upgrading-cli.adoc#L180